cognite_toolkit/_cdf_tk/commands/build_v2/build_issues.py

```diff
@@ -0,0 +1,45 @@
+import sys
+
+if sys.version_info >= (3, 11):
+    from typing import Self
+else:
+    from typing_extensions import Self
+
+from pydantic import BaseModel, RootModel
+
+from cognite_toolkit._cdf_tk.tk_warnings import ToolkitWarning, WarningList
+
+
+class BuildIssue(BaseModel):
+    """Issue with the build. Can have a recommendation for the user to improve the build."""
+
+    description: str
+
+
+class BuildIssueList(RootModel[list[BuildIssue]]):
```

Comment:
I think if you subclass UserList you do not have to repeat all the standard list method (append, extend, __len__)